runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/ComputationState.java

```diff
@@ -120,8 +121,9 @@ private void forceExecute(Work work) {
   }
 
   /** Adds any work started before the refreshDeadline to the GetDataRequest builder. */
-  public List<Windmill.KeyedGetDataRequest> getKeysToRefresh(Instant refreshDeadline) {
-    return activeWorkState.getKeysToRefresh(refreshDeadline);
+  public List<Windmill.KeyedGetDataRequest> getKeysToRefresh(
```

Comment:
If list shouldn't be mutated by caller, use `ImmutableList` go/java-practices/choosing-collections#immutable

Reply:
[5dfe51f](https://github.com/apache/beam/pull/29592/commits/5dfe51fe62511b141b0395536ca7698a94362bdc)